Package.swift

```diff
@@ -67,6 +67,6 @@ let package = Package(
                         .product(name: "SnapshotTesting", package: "swift-snapshot-testing")
                     ],
                     exclude: ["__Snapshots__"],
-                    resources: [.copy("Resources/image.png")])
+                    resources: [.copy("Resources/image_1.jpg"), .copy("Resources/image_2.jpg")])
```

Comment:
Unfortunately looks like this doesn't support wildcards.

Reply:
that's unfortunate. But... It's just swift code, though, right? So we could dynamically read the contents of the folder?

Reply:
Hmm I'll check that, yeah.

Reply:
Done.

Reply:
Actually this code:
```swift
/// Looks for resources for RevenueCatUI tests
/// Must look for them individually because using `.copy(directory)` fails to codesign.
private func paywallTestResources(_ file: StaticString = #file) -> [Resource] {
    let folderURL = URL(fileURLWithPath: String(describing: file))
        .deletingLastPathComponent()
        .appendingPathComponent("Tests")
        .appendingPathComponent("RevenueCatUITests")
        .appendingPathComponent("Resources")

    // swiftlint:disable:next force_try
    let fileURLs = try! FileManager.default.contentsOfDirectory(
        at: folderURL,
        includingPropertiesForKeys: nil,
        options: []
    )

    return fileURLs
        .filter { $0.pathExtension == "jpg" }
        .map { $0.absoluteString }
        .map {
            // Convert paths to the form "Resources/file.jpg"
            return String($0[$0.range(of: "Resources/")!.lowerBound...])
        }
        .map { .copy($0) }
}
```

Failed in CI:
```
Invalid manifest (compiled with: ["/Applications/Xcode-14.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc", "-vfsoverlay", "/var/folders/bl/wbxjgtzx7j5_mjsmfr3ynlc00000gp/T/TemporaryDirectory.OGVv7w/vfs.yaml", "-L", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-lPackageDescription", "-Xlinker", "-rpath", "-Xlinker", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-target", "x86_64-apple-macos12.0", "-sdk", "/Applications/Xcode-14.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk", "-swift-version", "5", "-I", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-sdk", "/Applications/Xcode-14.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk", "-package-description-version", "5.5.0", "-Xfrontend", "-serialize-diagnostics-path", "-Xfrontend", "/Users/distiller/Library/Caches/org.swift.swiftpm/manifests/ManifestLoading/purchases-ios.dia", "/Package.swift", "-disallow-use-new-driver", "-o", "/var/folders/bl/wbxjgtzx7j5_mjsmfr3ynlc00000gp/T/TemporaryDirectory.s0fhyy/purchases-ios-manifest"])
<unknown>:0: warning: legacy driver is now deprecated; consider avoiding specifying '-disallow-use-new-driver'main/Package.swift:85: Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=260 "The file “Resources” couldn’t be opened because there is no such file." UserInfo={NSURL=file:///Tests/RevenueCatUITests/Resources, NSFilePath=/Tests/RevenueCatUITests/Resources, NSUnderlyingError=0x600000605f20 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}} in https://github.com/RevenueCat/purchases-ios2023-07-21 05:16:34.966 xcodebuild[5345:14149] Writing error result bundle to /var/folders/bl/wbxjgtzx7j5_mjsmfr3ynlc00000gp/T/ResultBundle_2023-21-07_05-16-0034.xcresult
xcodebuild: error: Could not resolve package dependencies:
  Invalid manifest (compiled with: ["/Applications/Xcode-14.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc", "-vfsoverlay", "/var/folders/bl/wbxjgtzx7j5_mjsmfr3ynlc00000gp/T/TemporaryDirectory.OGVv7w/vfs.yaml", "-L", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-lPackageDescription", "-Xlinker", "-rpath", "-Xlinker", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-target", "x86_64-apple-macos12.0", "-sdk", "/Applications/Xcode-14.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk", "-swift-version", "5", "-I", "/Applications/Xcode-14.3.1.app/Contents/PlugIns/IDESwiftPackageCore.framework/Versions/A/Frameworks/SwiftPM.framework/SharedSupport/ManifestAPI", "-sdk", "/Applications/Xcode-14.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk", "-package-description-version", "5.5.0", "-Xfrontend", "-serialize-diagnostics-path", "-Xfrontend", "/Users/distiller/Library/Caches/org.swift.swiftpm/manifests/ManifestLoading/purchases-ios.dia", "/Package.swift", "-disallow-use-new-driver", "-o", "/var/folders/bl/wbxjgtzx7j5_mjsmfr3ynlc00000gp/T/TemporaryDirectory.s0fhyy/purchases-ios-manifest"])
<unknown>:0: warning: legacy driver is now deprecated; consider avoiding specifying '-disallow-use-new-driver'main/Package.swift:85: Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=260 "The file “Resources” couldn’t be opened because there is no such file." UserInfo={NSURL=file:///Tests/RevenueCatUITests/Resources, NSFilePath=/Tests/RevenueCatUITests/Resources, NSUnderlyingError=0x600000605f20 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}} in https://github.com/RevenueCat/purchases-ios
[05:16:36]: Exit status: 74
```

It's probably not worth it TBH for just a few files.